packages/pages/package.json

```diff
@@ -115,43 +113,32 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
   },
-  "babel": {
-    "presets": [
-      [
-        "@babel/preset-env",
-        {
-          "targets": {
-            "node": "current"
-          }
-        }
-      ],
-      "@babel/preset-typescript"
-    ]
-  },
   "jest": {
     "rootDir": "..",
     "collectCoverageFrom": [
-      "<rootDir>/pages/src/**/*.{ts,js}"
+      "<rootDir>/pages/src/**/*.{ts,tsx,js}"
     ],
     "globals": {
       "ts-jest": {
         "tsconfig": {
           "allowSyntheticDefaultImports": true,
           "esModuleInterop": true,
           "allowJs": true,
-          "target": "ESNext"
+          "target": "ESNext",
+          "jsx": "react"
         }
       }
     },
     "verbose": true,
     "testMatch": [
-      "<rootDir>/pages/src/**/*.test.{ts,js}"
+      "<rootDir>/pages/src/**/*.test.{ts,tsx,js}"
     ],
     "transform": {
       "\\.[jt]sx?$": "ts-jest"
     },
     "moduleNameMapper": {
       "^(.+)\\.js$": "$1"
-    }
+    },
+    "testEnvironment": "jsdom"
```

Comment:
I don't think we want to globally change this. The default test env is node. If you need jsdom for your specific test then you can override it for your particular test suite. See `packages/pages/src/util/runtime.test.ts`.